perf/bonnie.py

```diff
@@ -60,6 +64,16 @@ def setUp(self):
         process.run('./configure')
         build.make(self.srcdir)
 
+        part_obj = Partition(self.disk, mountpoint=self.scratch_dir)
+        self.log.info("Test will run on %s" % self.scratch_dir)
+        self.log.info("Unmounting the disk/dir before creating file system")
+        part_obj.unmount()
+        self.log.info("creating file system")
+        part_obj.mkfs(self.fstype)
+        self.log.info("Mounting disk %s on directory %s",
+                      self.disk, self.scratch_dir)
+        part_obj.mount()
+
```

Comment:
can we need to unmount the file device in tearDown section as it always  good clean up after test ?